sematic/ui/src/types/Types.tsx

```diff
@@ -691,6 +695,24 @@ function LinkValueView(props: ValueViewProps) {
   );
 }
 
+function DatetimeValueView(props: ValueViewProps) {
+  let {valueSummary} = props;
+  let {values} = valueSummary;
+  console.log(values);
+  return (
+      <DateTimePicker
```

Comment:
Instead of a "picker," which is probably best suited for editable fields, is there a good component optimized for date time displays we could use instead?

Reply:
I am not sure if there is something in `MUI` that would nicely show the datetime (I have found just the pickers before). I think that it should be enough to just show the date with time, and for that, I removed the date picker and use simple Typography. (I can agree that date picker just for visualization is a bit of overkill). See https://github.com/sematic-ai/sematic/pull/220/commits/b44b5052cd6cc9cbd07ce4c233259a70d17c12de.

Anyway, I am open to suggestions here 🙂

Reply:
**Currently it looks like this:** 

<img width="826" alt="Screenshot 2022-11-05 at 13 03 26" src="https://user-images.githubusercontent.com/32076170/200119276-c51c9abd-eb01-499f-94cc-4358076c8338.png">